src/cascadia/Remoting/Monarch.cpp

```diff
@@ -1106,8 +1110,9 @@ namespace winrt::Microsoft::Terminal::Remoting::implementation
             // In the case where window couldn't be found, then create a window
             // for that name / ID.
             //
-            // Don't let the window literally be named "-1", because that's silly
-            auto request = winrt::make_self<implementation::WindowRequestedArgs>(window == L"-1" ? L"" : window,
+            // Don't let the window literally be named "-1", because that's silly. Same with "new"
+            const bool nameIsReserved = window == L"-1" || window == L"new";

```

Comment:
should we just use -1 in the action instead of the magic word "new"?

fwiw: is `new` _already_ a magic word? if not, have we accidentally made it one?

Reply:
`AppLogic::_doFindTargetWindow` has it as a magic word. 

[Windows Terminal Session Management # Scenario: Run wt in the current window](https://github.com/microsoft/terminal/blob/main/doc/specs/%235000%20-%20Process%20Model%202.0/%234472%20-%20Windows%20Terminal%20Session%20Management.md#scenario-run-wt-in-the-current-window)

> If `window-id` is a negative number, or the reserved name `new`, run the commands in a new Terminal window.

\</discuss>

Reply:
Fair. I will ask, then: why do we need to special case it in more than one place? Shouldn't all window _finding and creation_ go through the same funnel?

Reply:
Boy wouldn't that be nice. As it is, finding a window depends on knowing what the glomming setting is, so that needs to have access to the settings, whereas creation doesn't so much. So finding can't get out of TerminalApp.

Could creation / move to be merged in with find? maybe?